patches/@clerk__astro@1.5.2.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/package.json b/package.json
+index ecf86cb4c8cf86d6fc7d6455d9dbca69ff8b0eea..075f1f5e7fe115dfb9bda50d1ccfe4eb616c15f3 100644
+--- a/package.json
++++ b/package.json
+@@ -29,9 +29,9 @@
+   "imports": {
+     "#async-local-storage": {
+       "workerd": "./dist/async-local-storage.server.js",
+-      "browser": "./dist/async-local-storage.client.js",
+       "node": "./dist/async-local-storage.server.js",
+-      "default": "./dist/async-local-storage.server.js"
++      "default": "./dist/async-local-storage.server.js",
++      "browser": "./dist/async-local-storage.client.js"
+     }
+   },
+   "exports": {
+@@ -83,7 +83,7 @@
+     "@clerk/eslint-config-custom": "0.0.2"
+   },
+   "peerDependencies": {
+-    "astro": "^3.2.0 || ^4.0.0"
++    "astro": "^4.15.0 || ^5.0.0"
```

Comment:
If we plan on keeping v4, we should require `4.15.0` at least to have the [view transitions helper](https://astro.build/blog/astro-4150/#swap-helpers-for-viewtransitions) available. We use that [here](https://github.com/clerk/javascript/blob/main/packages/astro/src/internal/swap-document.ts)